### PR TITLE
1 add eslint rules and prettier for server

### LINKED ISSUE #1: Add eslint rules and prettier for server
* add eslint and basic config for server
* add prettier and basic config for server
* dont forget to add npm scripts for linting and formatting

### PR DESCRIPTION
**Pull request by celikmurat1845**
Closes #1 
- add eslint configuration files
- add prettier configuration files

### CODE REVIEW

**Review by metehancelik — APPROVED** [review]

**Review by metehancelik — CHANGES_REQUESTED** [review]
add npm scripts

File: apps/server/package.json [hunk 1]
```diff
@@ -13,7 +13,17 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "ts-node-dev": "^2.0.0"
+    "@typescript-eslint/eslint-plugin": "^6.7.0",
+    "@typescript-eslint/parser": "^6.7.0",
+    "eslint": "^8.49.0",
+    "eslint-config-prettier": "^9.0.0",
+    "eslint-config-standard-with-typescript": "^39.0.0",
```
Comment (demirtasdurmus) [inline apps/server/package.json]: @celikmurat1845, it seems you are not using plugins at line 20,21,22,23. I am confused, if you want to use them you should add them to your config file, else please remove them.

File: apps/server/package.json [hunk 2]
```diff
@@ -13,7 +13,17 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "ts-node-dev": "^2.0.0"
+    "@typescript-eslint/eslint-plugin": "^6.7.0",
```
Comment (demirtasdurmus) [inline apps/server/package.json]: And please add npm script to lint in your scripts, then we can have a chance to check if linting rules really do work.

**Review by demirtasdurmus — CHANGES_REQUESTED** [review]

File: apps/server/.eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+  "root": true,
+  "env": {
+    "es2021": true,
+    "node": true
```
Comment (demirtasdurmus) [inline apps/server/.eslintrc.json:5]: test

**Review by demirtasdurmus — APPROVED** [review]

**Review by metehancelik — APPROVED** [review]

**Pull request merged**